generator/transit_generator.hpp

```diff
@@ -102,11 +103,66 @@ class DeserializerFromJson
   OsmIdToFeatureIdsMap const & m_osmIdToFeatureIds;
 };
 
+/// \brief the data contains all the information to make TRANSIT_FILE_TAG section.
+class GraphData
+{
+public:
+  // @todo(bykoianko) All the methods below should be rewrite with the help of visitors.
```

Comment:
rewritten?

Reply:
Yes. The todo removed since the visitors are written.